src/features/Mission/Mission.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable comma-dangle */
```

Comment:
[OPTIONAL] It would be nice if you could fix linter errors without this kind of comment /* eslint-disable ... */ because this type of comment is used in very few cases, and in this case, you can solve such errors easily. Having experience in solving all sorts of linter errors will allow you to write clean code without effort, which will increase your skills as a software developer 😃